From 3c109673dfbc730b926d536dffc0f58f92513041 Mon Sep 17 00:00:00 2001
From: Matt Selsky <matthew.selsky@twosigma.com>
Date: Wed, 30 Nov 2016 01:51:19 -0500
Subject: [PATCH 079/268] Fix spelling and asciidoc markup

Make sure that arguments to short options are italicized if their long option
counterparts are italicized.
---
 docs/assoc.txt                  |  2 +-
 docs/cluster.txt                |  2 +-
 docs/decode.txt                 |  4 ++--
 docs/discover.txt               |  2 +-
 docs/driver_hpgps.txt           |  2 +-
 docs/includes/mon-commands.txt  |  2 +-
 docs/includes/ntpd-body.txt     | 30 +++++++++++++++---------------
 docs/includes/ntpdig-body.txt   | 22 +++++++++++-----------
 docs/includes/ntpfrob-body.txt  |  4 ++--
 docs/includes/ntptrace-body.txt |  2 +-
 docs/includes/ntpviz-body.txt   |  2 +-
 docs/includes/ntpwait-body.txt  |  4 ++--
 docs/index.txt                  |  4 ++--
 docs/mode6.txt                  |  2 +-
 docs/ntpspeak.txt               |  6 +++---
 docs/quick.txt                  |  4 ++--
 docs/statfiles.txt              |  4 ++--
 17 files changed, 49 insertions(+), 49 deletions(-)

diff --git a/docs/assoc.txt b/docs/assoc.txt
index ee14ed7..7b413a3 100644
--- a/docs/assoc.txt
+++ b/docs/assoc.txt
@@ -112,7 +112,7 @@ includes only partial support for it. The deprecated +peer+ directive
 which formerly set up a symmetric active association is now a synonym
 for +server+. Servers which receive symmetric active messages will
 immediately reply with symmetric passive responses without setting up
-any new association; essientially they treat such messages exactly
+any new association; essentially they treat such messages exactly
 like client-mode messages aside from putting a different mode number
 into the response.
 
diff --git a/docs/cluster.txt b/docs/cluster.txt
index bc6da35..fb27fa4 100644
--- a/docs/cluster.txt
+++ b/docs/cluster.txt
@@ -56,7 +56,7 @@ Let φ~max~ be the maximum select jitter and φ~min~ be the minimum
 peer jitter over all candidates on the list. In (a), candidate 1 has the
 highest select jitter, so φ~_max_~ = φ~S~(1). Candidate 4 has the lowest
 peer jitter, so φ~min~ = φ~R~(4). Since φ~max~ > φ~min~, select
-jitter dominates peer jitter,the algorithm prunes candidate 1. In (b),
+jitter dominates peer jitter, the algorithm prunes candidate 1. In (b),
 φ~max~ = φ~S~(3) and φ~min~ =φ~R~(4). Since φ~max~ < φ~min~,
 pruning additional candidates does not reduce select jitter, the
 algorithm terminates with candidates 2, 3 and 4 as survivors.
diff --git a/docs/decode.txt b/docs/decode.txt
index 8d2f8ac..438f6be 100644
--- a/docs/decode.txt
+++ b/docs/decode.txt
@@ -47,7 +47,7 @@ In addition, there is a set of clock events for each association that
 supports a reference clock. Events are normally reported to the
 +protostats+ monitoring file and optionally to the system log.
 //The now-removed trap feature was mentioned here - we're marking
-//this spot in case sometning like it is reintroduced.
+//this spot in case something like it is reintroduced.
 
 In the following tables the Event Field is the status or event code
 assigned and the Message Field a short string used for display and event
@@ -253,7 +253,7 @@ Not all bits are in use, but have been kept for backwards compatibility.
 
 Kiss codes are used in kiss-o'-death (KoD) packets, billboard displays
 and log messages. They consist of a string of four zero-padded ASCII
-charactes. In practice they are informal and tend to change with time
+characters. In practice they are informal and tend to change with time
 and implementation. Some of these codes can appear in the reference
 identifier field in +ntpq+ billboards. Following is the current list:
 
diff --git a/docs/discover.txt b/docs/discover.txt
index 9542ab7..e419568 100644
--- a/docs/discover.txt
+++ b/docs/discover.txt
@@ -127,7 +127,7 @@ RFC-2770.
 A multicast server is configured using the +broadcast+ command, but
 specifying a multicast address instead of a broadcast address. Note
 that there is a subtle distinction between the IPv4 and IPv6 address
-families. The IPv4 broadcast or mulitcast mode is determined by the
+families. The IPv4 broadcast or multicast mode is determined by the
 IPv4 class. For IPv6 the same distinction can be made using the
 link-local prefix FF02 for each interface and site-local prefix FF05
 for all interfaces.
diff --git a/docs/driver_hpgps.txt b/docs/driver_hpgps.txt
index b9578fa..53ae5af 100644
--- a/docs/driver_hpgps.txt
+++ b/docs/driver_hpgps.txt
@@ -17,7 +17,7 @@ Two-unit pairs of these are sometimes known as a KS-24361.
 These receivers are very old designs, dating from around 1990, so
 their GPS receiver stages are not very sensitive by modern
 standards. On the other hand, lots of them are available cheap,
-because they used to be standard kit on cell towers and were surplused
+because they used to be standard kit on cell towers and were surplussed
 off in the thousands.
 
 They use HP SmartClock (TM) to implement an Enhanced GPS receiver. The
diff --git a/docs/includes/mon-commands.txt b/docs/includes/mon-commands.txt
index e026c25..9725e7c 100644
--- a/docs/includes/mon-commands.txt
+++ b/docs/includes/mon-commands.txt
@@ -227,7 +227,7 @@ generated line.
 |+0+        |#        |ru_inblock: file blocks in
 |+2328+     |#        |ru_oublock: file blocks out
 |+64226+    |#        |ru_nvcsw: context switches, wait
-|+1+        |#        |ru_nivcsw: context switches, prempts
+|+1+        |#        |ru_nivcsw: context switches, preempts
 |+0+        |#        |ru_nsignals: signals
 |+4308+     |#        |ru_maxrss: resident set size, kilobytes
 |==================================================
diff --git a/docs/includes/ntpd-body.txt b/docs/includes/ntpd-body.txt
index 5a8c2dd..c1672d3 100644
--- a/docs/includes/ntpd-body.txt
+++ b/docs/includes/ntpd-body.txt
@@ -94,7 +94,7 @@ never a good idea.
 +-b+, +--bcastsync+::
   Allow us to sync to broadcast servers.
 
-+-c+ string, +--configfile+=_string_::
++-c+ _string_, +--configfile+=_string_::
   configuration file name.
 +
 The name and path of the configuration file, +/etc/{ntpconf}+ by
@@ -104,11 +104,11 @@ default.
   Increase debug verbosity level. This option may appear an unlimited
   number of times.
 
-+-D+ number, +--set-debug-level+=_number_::
++-D+ _number_, +--set-debug-level+=_number_::
   Set the debug verbosity level. This option may appear an unlimited
   number of times. This option takes an integer number as its argument.
 
-+-f+ string, +--driftfile+=_string_::
++-f+ _string_, +--driftfile+=_string_::
   frequency drift file name.
 +
 The name and path of the frequency file, +/etc/ntp.drift+ by default.
@@ -141,7 +141,7 @@ cause the time to be stepped back so this option must not be used if
 applications requiring monotonic time are running. See the _tinker_
 configuration file directive for other options.
 
-+-i+ string, +--jaildir+=_string_::
++-i+ _string_, +--jaildir+=_string_::
   Jail directory.
 +
 Chroot the server to the directory _jaildir_ This option also implies
@@ -150,7 +150,7 @@ need to also specify a +-u+ option. This option is only available if
 the OS supports adjusting the clock without full root privileges. This
 option is supported under Linux, NetBSD and Solaris.
 
-+-I+ iface, +--interface+=_iface_::
++-I+ _iface_, +--interface+=_iface_::
   Listen on an interface name or address. This option may appear an
   unlimited number of times.
 +
@@ -160,14 +160,14 @@ option also implies not opening other addresses, except wildcard and
 localhost. This option is deprecated. Please consider using the
 configuration file _interface_ command, which is more versatile.
 
-+-k+ string, +--keyfile+=_string_::
++-k+ _string_, +--keyfile+=_string_::
   path to symmetric keys.
 +
 Specify the name and path of the symmetric key file. +/etc/ntp.keys+
 is the default. This is the same operation as the _keys_
 configuration file directive.
 
-+-l+ string, +--logfile+=_string_::
++-l+ _string_, +--logfile+=_string_::
   path to the log file.
 +
 Specify the name and path of the log file. The default is the system
@@ -197,14 +197,14 @@ server to be discovered via mDNS client lookup.
 To the extent permitted by the operating system, run +ntpd+ at the
 highest priority.
 
-+-p+ string, +--pidfile+=_string_::
++-p+ _string_, +--pidfile+=_string_::
   path to the PID file.
 +
 Specify the name and path of the file used to record +ntpd+'s process
 ID. This is the same operation as the _pidfile_
 configuration file directive.
 
-+-P+ number, +--priority+=_number_::
++-P+ _number_, +--priority+=_number_::
   Process priority. This option takes an integer number as its argument.
 +
 To the extent permitted by the operating system, run +ntpd+ at the
@@ -220,34 +220,34 @@ which has been replaced with a shell script. The +-g+ and +-x+ options can be
 used with this option. Note: The kernel time discipline is disabled
 with this option.
 
-+-r+ string, +--propagationdelay+=_string_::
++-r+ _string_, +--propagationdelay+=_string_::
   Broadcast/propagation delay.
 +
 Specify the default propagation delay from the broadcast/multicast
 server to this client. This is necessary only if the delay cannot be
 computed automatically by the protocol.
 
-+-s+ string, +--statsdir+=_string_::
++-s+ _string_, +--statsdir+=_string_::
   Statistics file location.
 +
 Specify the directory path for files created by the statistics
 facility. This is the same operation as the _statsdir_
 configuration file directive.
 
-+-t+ tkey, +--trustedkey+=_tkey_::
++-t+ _tkey_, +--trustedkey+=_tkey_::
   Trusted key number. This option may appear an unlimited number of
   times.
 +
 Add the specified key number to the trusted key list.
 
-+-u+ string, +--user+=_string_::
++-u+ _string_, +--user+=_string_::
   Run as userid (or userid:groupid).
 +
 Specify a user, and optionally a group, to switch to. This option is
 only available if the OS supports adjusting the clock without full
 root privileges. This option is supported under Linux, NetBSD and Solaris.
 
-+-U+ number, +--updateinterval+=_number_::
++-U+ _number_, +--updateinterval+=_number_::
   interval in seconds between scans for new or dropped interfaces. This
   option takes an integer number as its argument.
 +
@@ -257,7 +257,7 @@ performed shortly after the interface change has been detected by the
 system. Use 0 to disable scanning. 60 seconds is the minimum time
 between scans.
 
-+-w+ number, +--wait-sync+=_number_::
++-w+ _number_, +--wait-sync+=_number_::
   Seconds to wait for first clock sync. This option must not appear in
   combination with any of the following options: nofork, quit.
   This option takes an integer number as its argument.
diff --git a/docs/includes/ntpdig-body.txt b/docs/includes/ntpdig-body.txt
index b3e91ab..e49f366 100644
--- a/docs/includes/ntpdig-body.txt
+++ b/docs/includes/ntpdig-body.txt
@@ -75,7 +75,7 @@ to the IPv4 namespace.
 Force DNS resolution of the following host names on the command line
 to the IPv6 namespace.
 
-+-a+ auth-keynumber, +--authentication+=_auth-keynumber_::
++-a+ _auth-keynumber_, +--authentication+=_auth-keynumber_::
   Enable authentication with the key _auth-keynumber_. This option takes
   an integer number as its argument.
   +
@@ -84,7 +84,7 @@ argument. The argument of this option is the _keyid_, a number
 specified in the _keyfile_ as this key's identifier. See the _keyfile_
 option (+-k+) for more details.
 
-+-b+ broadcast-address, +--broadcast+=_broadcast-address_::
++-b+ _broadcast-address_, +--broadcast+=_broadcast-address_::
   Listen to the address specified for broadcast time sync. This option
   may appear an unlimited number of times.
 +
@@ -92,7 +92,7 @@ If specified +ntpdig+ will listen to the specified address for NTP
 broadcasts. The default maximum wait time can (and probably should) be
 modified with +-t+.
 
-+-c+ host-name, +--concurrent+=_host-name_::
++-c+ _host-name_, +--concurrent+=_host-name_::
   Concurrently query all IPs returned for host-name. This option may
   appear an unlimited number of times.
 +
@@ -110,11 +110,11 @@ send concurrent queries.  This is appropriate when using a server pool.
   Increase debug verbosity level. This option may appear an unlimited
   number of times.
 
-+-D+ number, +--set-debug-level+=_number_::
++-D+ _number_, +--set-debug-level+=_number_::
   Set the debug verbosity level. This option may appear an unlimited
   number of times. This option takes an integer number as its argument.
 
-+-g+ milliseconds, +--gap+=_milliseconds_::
++-g+ _milliseconds_, +--gap+=_milliseconds_::
   The gap (in milliseconds) between time requests. This option takes an
   integer number as its argument. The default _milliseconds_ for this
   option is 50.
@@ -129,7 +129,7 @@ response if the first source attempted is slow or unreachable.
 +-j+::
   Output to stdout in JSON, suppressing syslog messages.
 
-+-K+ file-name, +--kod+=_file-name_::
++-K+ _file-name_, +--kod+=_file-name_::
   KoD history filename. The default _file-name_ for this option is:
   +/var/db/ntp-kod+.
 +
@@ -137,20 +137,20 @@ Specifies the filename to be used for the persistent history of KoD
 responses received from servers. If the file does not exist, a warning
 message will be displayed. The file will not be created.
 
-+-k+ file-name, +--keyfile+=_file-name_::
++-k+ _file-name_, +--keyfile+=_file-name_::
   Look in this file for the key specified with +-a+.
 +
 This option specifies the keyfile. +ntpdig+ will search for the key
 specified with +-a+ keyno in this file. See _{ntpkeysman}_ for more
 information.
 
-+-l+ file-name, +--logfile+=_file-name_::
++-l+ _file-name_, +--logfile+=_file-name_::
   Log to specified logfile.
 +
 This option causes the client to write log messages to the specified
 _logfile_.
 
-+-M+ number, +--steplimit+=_number_::
++-M+ _number_, +--steplimit+=_number_::
   Adjustments less than _steplimit_ milliseconds will be slewed. This option
   takes an integer number as its argument. The value of _number_ is
   constrained to being greater than or equal to 0,
@@ -161,7 +161,7 @@ _clock_settime()_ or local equivalent. The default value is 0, which
 means all adjustments will be stepped. This is a feature, as different
 situations demand different values.
 
-+-o+ number, +--ntpversion+=_number_::
++-o+ _number_, +--ntpversion+=_number_::
   Send _number_ as our NTP protocol version. This option takes an integer
   number as its argument.  Defaults to 4.  Seldom useful.
 
@@ -186,7 +186,7 @@ situations demand different values.
   enough to accomplish the required offset (phase) correction. This
   typically requires +ntpdig+ be invoked as the superuser ("root").
 
-+-t+ seconds, +--timeout+=_seconds_::
++-t+ _seconds_, +--timeout+=_seconds_::
   The number of seconds to wait for responses. This option takes an
   integer number as its argument. The default _seconds_ for this option
   is: 5.
diff --git a/docs/includes/ntpfrob-body.txt b/docs/includes/ntpfrob-body.txt
index 3f27253..6f1db9b 100644
--- a/docs/includes/ntpfrob-body.txt
+++ b/docs/includes/ntpfrob-body.txt
@@ -37,9 +37,9 @@ utilities.
   Raw mode.  Only applies to the jitter mode, and means the raw
   clock samples should be emitted to stdout for postanalysis.
 +-?+::
-  Print sueage and exit.
+  Print usage and exit.
 +-h+::
-  Print sueage and exit.
+  Print usage and exit.
 
 [[usage]]
 == Usage ==
diff --git a/docs/includes/ntptrace-body.txt b/docs/includes/ntptrace-body.txt
index a250bd3..c14a76c 100644
--- a/docs/includes/ntptrace-body.txt
+++ b/docs/includes/ntptrace-body.txt
@@ -20,7 +20,7 @@ For +ntptrace+ to work properly, each of these servers must
 implement the NTP Control and Monitoring Protocol specified in RFC 1305
 and enable NTP Mode 6 control packets.  Nowadays it is usual for
 public timeservers to disable Mode 6 queries, so this script is
-unlikely to be very useful unless you have a collectioon of
+unlikely to be very useful unless you have a collection of
 specially-configured timeservers on your LAN.
 
 If given no arguments, ntptrace starts with localhost. Here is an
diff --git a/docs/includes/ntpviz-body.txt b/docs/includes/ntpviz-body.txt
index f130f6a..5897069 100644
--- a/docs/includes/ntpviz-body.txt
+++ b/docs/includes/ntpviz-body.txt
@@ -85,7 +85,7 @@ subset of comparative plots for multiple directories.
     (1388x768).  'l' for large screens (1920x1080).  'm' is the default.
 
 -D DLVL or --debug DLVL::
-    Set the debug level to DLVL.  Lsrger DLVL leads to more verbosity. +
+    Set the debug level to DLVL.  Larger DLVL leads to more verbosity. +
     0 is the default, quiet except for all ERRORs and some WARNINGs. +
     1 shows some environment info and basic program progress. +
     2 leaves the plot file in the system temp directory. +
diff --git a/docs/includes/ntpwait-body.txt b/docs/includes/ntpwait-body.txt
index 26106e8..e778b2f 100644
--- a/docs/includes/ntpwait-body.txt
+++ b/docs/includes/ntpwait-body.txt
@@ -16,14 +16,14 @@ yet produced a synchronized and stable system clock._
 
 == OPTIONS ==
 
-+-n+ number, +--tries+=_number_::
++-n+ _number_, +--tries+=_number_::
   Number of times to check ntpd. This option takes an integer number as
   its argument. The default _number_ for this option is: 100.
 +
 The maximum number of times we will check _ntpd_ to see if it has been
 able to synchronize and stabilize the system clock.
 
-+-s+ secs-between-tries, +--sleep+=_secs-between-tries_::
++-s+ _secs-between-tries_, +--sleep+=_secs-between-tries_::
   How long to sleep between tries. This option takes an integer number
   as its argument. The default _secs-between-tries_ for this option
   is: 6.
diff --git a/docs/index.txt b/docs/index.txt
index 0f853cd..d7acab7 100644
--- a/docs/index.txt
+++ b/docs/index.txt
@@ -89,7 +89,7 @@ few will be user-visible.
   into +ntpq+.
 
 * As noted above, Autokey is not supported; that code has been
-  removed, as it was chronically prone to security vulnerabilties.
+  removed, as it was chronically prone to security vulnerabilities.
 
 * peer mode has been removed.  The keyword peer in ntp.conf is now
   just an alias for keyword server.  Incoming peer packets are ignored.
@@ -295,7 +295,7 @@ link:warp.html[How NTP Works]::
   Gives an overview of the NTP daemon architecture and how it works.
 link:outside-tools.html[Outside Tools]::
   A variety of tools not directly maintained along with NTP can be
-  useful for adminiseting and monitoring it. This page collects some
+  useful for administering and monitoring it. This page collects some
   references and links.
 
 [[build]]
diff --git a/docs/mode6.txt b/docs/mode6.txt
index d39ee2f..ef7517c 100644
--- a/docs/mode6.txt
+++ b/docs/mode6.txt
@@ -391,7 +391,7 @@ the stanza.
 Attributes within each stanza are deliberately issued in a random
 order, and ntpd occasionally issues an attribute with a
 randomly-generated name and value. This is an attempt to prevent Mode
-6 clients from making brittle assumptioons about the inventory of
+6 clients from making brittle assumptions about the inventory of
 attributes and their transmission order.
 
 Clients can rely on all *.0 values preceding any *.1 values, that is
diff --git a/docs/ntpspeak.txt b/docs/ntpspeak.txt
index 20a2aec..75fa5e7 100644
--- a/docs/ntpspeak.txt
+++ b/docs/ntpspeak.txt
@@ -73,7 +73,7 @@
    Global Positioning System; also, "a GPS" is a radio receiver
    designed to get position and time fixes from the system. GPS fixes
    are derived from spherical trigonometry using the precisely known
-   positions of satellities in a geocentric coordinate system. GPS
+   positions of satellites in a geocentric coordinate system. GPS
    also provides time service; those that emit <<PPS>> are suitable
    as clock sources for Stratum 1 timeservers.
 
@@ -170,7 +170,7 @@
 
 [[popcorn spike]] popcorn spike::
   A packet from a particular peer that has a computed theta more than
-  three standard deviations from the prevous sample.  These are
+  three standard deviations from the previous sample.  These are
   normally treated as random outliers and suppressed.
 
 [[PPS]] PPS::
@@ -207,7 +207,7 @@
   with a refclock is implicitly Stratum 1.
 
 [[stratum]] stratum::
-  A "stratum" is a layer in the hieratchy of time servers.  a
+  A "stratum" is a layer in the hierarchy of time servers.  a
   <<refclock>> is considered stratum 0; a computer directly attached to
   a refclock is stratum 1; and a client served by a stratum N is
   stratum N+1. Often capitalized, especially when referring to all
diff --git a/docs/quick.txt b/docs/quick.txt
index 8795f90..bc02c0c 100644
--- a/docs/quick.txt
+++ b/docs/quick.txt
@@ -162,8 +162,8 @@ analogous group of servers by pinging them.
 Ideally, one would like one's servers to use multiple different kinds
 of timesources (as opposed to, say, all being GPS-based) and be split
 across different autonomous networks as a hedge against outages and
-routing probems.  Unfortunately, the random nature of pool allocation
-makes this imposssible to guarantee. It is, however, worth keeping in mind if
+routing problems.  Unfortunately, the random nature of pool allocation
+makes this impossible to guarantee. It is, however, worth keeping in mind if
 you can set up a custom configuration with non-pool servers that you
 have permission to use.
 
diff --git a/docs/statfiles.txt b/docs/statfiles.txt
index d60d146..7850d31 100644
--- a/docs/statfiles.txt
+++ b/docs/statfiles.txt
@@ -70,7 +70,7 @@ makes it too difficult to make it a standard process.  There are
 examples of programs to log data from different temperature sensors in
 the contrib/ directory.
 
-When this file exists, the ntpvi program will graph the data.
+When this file exists, the ntpviz program will graph the data.
 
 Each line in this file is one valid logged temperature.  The file can
 contain data from several different thermometers.  Each line is tagged
@@ -81,7 +81,7 @@ with the data source symbolic name in field 2.
 
   1471625701           Unix time, seconds since the start of the epoch
   TEMPER               Name of the temperature source.
-  24.9                 Meassured temperature in degree Celsius
+  24.9                 Measured temperature in degree Celsius
 ----------------------------------------------------------------------------
 
 //end
-- 
2.7.4

